tests/load-tests/run-max-concurrency.sh

```diff
@@ -42,7 +42,7 @@ else
     IFS=" " read -r -a maxConcurrencySteps <<<"${MAX_CONCURRENCY_STEPS:-1 5 10 25 50 100 150 200}"
     maxThreads=${MAX_THREADS:-10}
     threshold=${THRESHOLD:-300}
-    echo '{"maxThreads": '"$maxThreads"', "maxConcurrencySteps": "'"${maxConcurrencySteps[*]}"'", "threshold": '"$threshold"', "maxConcurrencyReached": 0}' | jq >"$output"
+    echo '{"startTimestamp":"'$(date +%FT%T%:z)'", "maxThreads": '"$maxThreads"', "maxConcurrencySteps": "'"${maxConcurrencySteps[*]}"'", "threshold": '"$threshold"', "maxConcurrencyReached": 0, "endTimestamp": ""}' | jq >"$output"
```

Comment:
Hmm, did not knew this `:z` format. I like to use `date -Iseconds --utc` but I guess yours works on Macs as well.

Reply:
In normal JSON we have just `timestamp` (without `start...` part), but I think this is better.

Reply:
With the `:z` I tried to match the timestamp format of the base JSON that is currently timezoned the same way.

As for the `timestamp`->`startTimestamp` we should make those fields consistent in all places (in a different PR), I agree, and IMHO `startTimestamp` is clearer what the timestamp is actually about.

Reply:
thx for the feedback